test/unit/logic/test_data/multisignature.js

```diff
@@ -89,6 +90,97 @@ var validTransaction = {
 	},
 };
 
+var invalidAllSignaturesReadyFalse = {
+	id: '10004093306508192097',
+	height: 2967,
+	blockId: '16880210663552206127',
+	type: 4,
+	timestamp: 39547828,
+	senderPublicKey:
+		'c094ebee7ec0c50ebee32918655e089f6e1a604b83bcaa760293c61e0f18ab6f',
+	senderId: '16313739661670634666L',
+	recipientId: null,
+	recipientPublicKey: null,
+	amount: 0,
+	fee: 1500000000,
+	signature:
+		'c66a726defca0fcdb5978292e4c999d37ba08dfa9ea0796de04e57f48f77489f3868754fc208c42ac957e259494040e61b16ee7a8d715eb198bedf963dc18907',
+	signatures: [
+		'02eee0660459c36916c3c230e48cd7bec84b9ebe30049202a85d950bd36988ed46f313bc43b8240bd3886d4eb0571253d6615aae14df0a97cf8d5420f491aa0a',
+		'a77e0f0a6e3db16542cf26268070a1a5bb69f6b90e855943c9cf8f3cde22c6c10e43e8443b33722973ebe7de6f6abcfb1792cd50c5082c66805c5ad9c486c108',
+	],
+	asset: {
+		multisignature: {
+			min: 2,
+			lifetime: 2,
+			keysgroup: [
+				'+bd6d0388dcc0b07ab2035689c60a78d3ebb27901c5a5ed9a07262eab1a2e9bd2',
+				'+addb0e15a44b0fdc6ff291be28d8c98f5551d0cd9218d749e30ddb87c6e31ca9',
+			],
+		},
+	},
+	ready: false,
+};
+
+var invalidSomeSignaturesReadyTrue = {
```

Comment:
Same with the rest.

Reply:
@diego-G it's invalid in the sense that if some signatures, but not all, are present and valid ready should be false. So if a transaction is sent with some valid signatures and ready: true it is an invalid payload but the code will fix it now.

Reply:
Sending an update for this one as the mock in did had all signatures (code follows the same path as for `invalidAllSignaturesReadyFalse ` so it's only the mock that's misleading)